ksql-engine/src/main/java/io/confluent/ksql/util/ExpressionTypeManager.java

```diff
@@ -46,6 +48,7 @@
 
   private final Schema schema;
   private final FunctionRegistry functionRegistry;
+  private final FunctionArguments functionArguments = new FunctionArguments();
 
   public ExpressionTypeManager(Schema schema, final FunctionRegistry functionRegistry) {
```

Comment:
I think it would be good to add a test case for this class to verify that we handle nested function calls correctly. Those don't seem to be exercised by the existing unit test.

Reply:
It is covered by `CodeGenRunnerTest#shouldHandleNestedUdfs` but there is no harm in adding one specifically in the unit test for this class